index.html

```diff
@@ -0,0 +1,114 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.0/css/all.min.css"
+      integrity="sha512-xh6O/CkQoPOWDdYTDqeRdPCVd1SpvCA9XXcUnZS2FmJNp1coAFzvtCN9BmamE+4aHK8yyUHUSCcJHgXloTyT2A=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="style.css" />
+
+    <title>conference - event</title>
+  </head>
+  <body>
+    <header>
+      <div class="upper-nav">
+        <ul>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Log out</a></li>
+        </ul>
+      </div>
+      <nav class="lower-nav">
+        <div class="title">
+         <a href="index.html"><h2 class="logo-h1">ICS<span class="logo-span">DA</h2></span></a> 
+        </div>
+        <ul class="lower-nav-ul">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li class="campaign-cdn"><a href="#">Campaign</a></li>
+        </ul>
+        <button class="nav-toggle">
+            <i id="open" class="fa-solid fa-bars"></i>
+            <i id="close" class="fa-solid fa-xmark"></i>
+        </button>
+      </nav>
+    </header>
+    <section class="hero-section">
+            <div class="hero-desc">    
+                <h2 class="hero-title">'Hello Data Science Enthusiasts'</h1>
+               <h1 class="hero-subtitle">International Conference On Statistics and Data Science</h1>
+               <div class="hero-para">
+                   <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Okap. </p>
+               </div>
+               <h2 class="date">2023.10.19(THU) ~20(FRI)</h2>
+               <h2 class="location">@Citadelle Laferrière in Okap, Palais Sans Souci and more</h2>
+               </div>
+    </section>
+    <!-- PROGRAMS SECTION -->
+    <section class="programs">
+      <h3 class="program-title">
```

Comment:
- [ ] Great job with the design. Please ensure that when the mobile menu popsup it shouldn't be possible to navigate through your work unless through it or when it is closed.
![Screenshot (459)_LI](https://user-images.githubusercontent.com/65298048/191358704-d8443bb7-a4be-4195-b2b2-a1019904566a.jpg)